src/Extensions/AddressExtensions.cs

```diff
@@ -9,5 +10,20 @@ public static class AddressExtensions
                 string.IsNullOrEmpty(value.SelectedAddress) : 
                 string.IsNullOrEmpty(value.AddressLine1) &&
                 string.IsNullOrEmpty(value.Postcode);
+
+        public static string ConvertAddressToTitleCase(this string address)
```

Comment:
should this be in string extensions not address extension?

Reply:
Just thought it was related to address hence when it was there. I can move to  the string extension